groot-btrfs-release/packages/groot-btrfs/pre_packaging

```diff
@@ -1,8 +1,11 @@
 #!/bin/bash
 set -e -x
 
-pushd $RELEASE_DIR/..
-source ./make/include/versioning.sh
+mkdir "${BUILD_DIR}/groot-btrfs/"
+
+pushd "$RELEASE_DIR/.."
+    source ./make/include/versioning.sh
+    cp -r * "${BUILD_DIR}/groot-btrfs/"
```

Comment:
I would prefer that the `cp -r * ...` would be replaced by a series of copies that mirror the now deleted list of patterns in the `spec` file:

```
mkdir "${BUILD_DIR}/groot-btrfs/base_image_puller"
cp -r base_image_puller/* "${BUILD_DIR}/groot-btrfs/base_image_puller"
...
```

This avoid including several subtrees that are not needed in the release package, e.g. `groot-btrfs-release`, `integration`, and `testhelpers`.